digest.go

```diff
@@ -241,19 +236,38 @@ func Digest(c *Context, open, closed []*PullRequest) error {
 		Open:   sortedOpen,
 		Closed: sortedClosed,
 	}
+
 	tmpl := template.Must(template.New("digest").Funcs(template.FuncMap{"markDown": markDowner}).Parse(htmlTemplate))
+	buf := new(bytes.Buffer)
+	if err := tmpl.Execute(buf, content); err != nil {
+		return err
+	}
+
+	options := premailer.NewOptions()
```

Comment:
Let's control the premailer through a command line flag. Put add an `--inline-styles` flag which defaults to true and `Context.InlineStyles bool`.